### ISSUE TITLE
Final touches 🧑‍🎨

### ISSUE THREAD

**Issue by dylanratcliffe** [overmindtech/cli#330]
https://github.com/overmindtech/cli/assets/8799341/b82be88f-230b-4e6f-99ab-3ced739df296

## Couple of issues

Once the plan is complete, the initial steps (Connected to Overmind etc.) are still present below the plan. Then when the revlink completes, they go away. They should never be there at all, just the "Planning Changes" and below

When we print the "✔︎ Calculating Risks" step, we print ~11 lines at once, this is too hard to track with your eyes and is jarring, maybe we should print *just* the calculating risks section, then the URL with a bit of a delay and a max of 1 empty line. Or potentially like this:

```
✔︎ Calculating Risks
   ? Mapping planned changes to current cloud resources
   + Processing changes (3 planned changes & 3 mapped resources)
   + Analyzing blast radius (17 affected resources & 28 relationships)
   + Returning enriched risks
    | Check the blast radius graph while you wait:
    | https://app.overmind.tech/changes/7777be50-007e-469a-9f20-82df8765b181/blast-radius
```

Another option could be to have "Saving" change to "Saved" and put the link on that same line after the saved

## Nice-to-have:

All of these ticks appear at the exact same time:

```
✔︎ Removing secrets
✔︎ Extracting 3 changing resources: 3 supported 0 unsupported
  ✔︎ aws_instance (2)
  ✔︎ aws_security_group (1)
✔︎ Uploading planned changes (new)
```

Which is quite jarring and kind of makes it hard to track. Would be good if we could add a small delay to each step here, even if just a few hundred milliseconds so your eyes can track them as they scroll up.
